python/ngraph/ops.py

```diff
@@ -951,6 +951,55 @@ def lrn(data,       # type: Node
     return LRN(data, alpha, beta, bias, size)
 
 
+@nameable_op
+def argmax(data,    # type: Node
+           axis,    # type: int
```

Comment:
Please provide a sane default for `axis` - 0 should work.
`etype` should not be required.